.github/.goreleaser_docker.yaml

```diff
@@ -0,0 +1,190 @@
+# Separate docker configuration to isolate docker dependency away from
```

Comment:
nit: I feel that it would be more apparent for all go-releaser things to be in the same spot. In this case the root of the repo.